### PR TITLE
Wool smuggling missions

### PR DESCRIPTION
**Pull request by Anarchist2**
## Summary
Adds a short mission about bringing some luxury clothing from New Kansas to Bloodsea.

### CODE REVIEW

File: data/human/human missions.txt [hunk 1]
```diff
@@ -4927,3 +4927,75 @@ mission "Riley"
...
+			choice
+				`	"Fine, I'll do it."`
+					goto accept
+				`	"That's not enough for me."`
+					goto decline
+			label accept
+			`	"Great! I'll call up the spaceport workers on <planet> to tell them to load the cargo onto your ship." The man walks away, leaving you to your own devices.`
+				accept
+			label decline
+			`	"Suit yourself, then." He hautily walks off into the spaceport.`
```
Comment (Hecter94) [inline data/human/human missions.txt:4978]:
```suggestion
			`	"Suit yourself, then." He haughtily walks off into the spaceport.`
```

File: data/human/human missions.txt [hunk 2]
```diff
@@ -4927,3 +4927,75 @@ mission "Riley"
...
+	cargo "wool garments" 15
+	source "New Kansas"
+	destination "Bloodsea"
+	to offer
+		has "Wool Smuggling 1: done"
+	on visit
+		dialog phrase "generic cargo on visit"
+	on complete
+		payment 200000
+		dialog `Your cargo of woollen garments is unloaded by several bulky workers, led by a one-armed woman. She hands you <payment> for the trouble of transport, then says, "Thanks for the shipment. I know that you captains think that we're all bad, but I'm glad that you were able to aid the more friendly among us.`
```
Comment (Hecter94) [inline data/human/human missions.txt:5000]:
```suggestion
		dialog `Your cargo of woollen garments is unloaded by several bulky workers led by a one-armed woman. She hands you <payment> for the trouble of transport, then says, "Thanks for the shipment. I know that you captains think that we're all bad, but I'm glad that you were able to aid the more friendly among us.`
```
Unnecessary comma

**Review by Hecter94 — CHANGES_REQUESTED** [review]

**Review by tibetiroka — APPROVED** [review]

**Review by Hecter94 — APPROVED** [review]

**Review by tibetiroka — APPROVED** [review]

File: data/human/human missions.txt [hunk 3]
```diff
@@ -4927,3 +4927,75 @@ mission "Riley"
...
 			`	You say your goodbyes, leaving to your ship and returning to the stars.`
 				decline
+
+
+
+mission "Wool Smuggling 1"
+	minor
+	name "Pickup at <planet>"
+	description "Head to <destination> to pick up a shipment of <tons> of woollen goods for delivery to Bloodsea."
+	cargo "(reserved)" 15
```
Comment (MasterOfGrey) [inline data/human/human missions.txt:4937]: Would it not be easier to put a block on the next mission and have this one not resolve until that one is offered?
That way people could still carry extra trade goods to New Kansas if they wanted to.

Comment (Anarchist2) [reply data/human/human missions.txt:4955]: That doesn't work; mission completion resolves before offering, and doesn't trigger again until you land/reload the save.

Comment (MasterOfGrey) [reply data/human/human missions.txt:4955]: So just have the next mission trigger on New Kansas if this one has `offered`.
Have the second mission give you a blocked message that you can't take it without clearing cargo - potentially have it make you visit the spaceport so you can repeat test it without taking off and landing again.
Then have accepting the 2nd mission fail the first.
The reward is on the second one after all.

File: data/human/human missions.txt [hunk 4]
```diff
@@ -4945,3 +4945,76 @@ mission "Riley"
...
+		distance "Muhlifain" 1 3
+	destination "New Kansas"
+	to offer
+		random < 45
+		"cargo space" >= 15
+		not "bloodsea joins free worlds"
+		not "bloodsea independent"
+	on offer
+		conversation
+			`As you walk around the spaceport, a stout man wearing a fine wool coat approaches you. "Captain, could you help me out with a delivery? I've been asking around the spaceport to get a shipment of woollen goods to another planet, but every merchant I've talked to so far has refused."`
```
Comment (Amazinite) [inline data/human/human missions.txt:4965]:
```suggestion
			`As you walk around the spaceport, a stout man wearing a fine wool coat approaches you. "Captain, could you help me out with a delivery? I've been asking around the spaceport to get a shipment of woolen goods to another planet, but every merchant I've talked to so far has refused."`
```

File: data/human/human missions.txt [hunk 5]
```diff
@@ -4945,3 +4945,76 @@ mission "Riley"
...
+		has "Wool Smuggling 1: done"
+	on offer
+		conversation
+			`When you land, dock workers begin loading crates with woollen garments on board as arranged. Time to set course for <planet>.`
+				accept
+	on visit
+		dialog phrase "generic cargo on visit"
+	on complete
+		payment 200000
+		dialog `Your cargo of woollen garments is unloaded by several bulky workers led by a one-armed woman. She hands you <payment> for the trouble of transport, then says, "Thanks for the shipment. I know that you captains think that we're all bad, but I'm glad that you were able to aid the more friendly among us.`
```
Comment (Amazinite) [inline data/human/human missions.txt:5019]:
```suggestion
		dialog `Your cargo of woolen garments is unloaded by several bulky workers led by a one-armed woman. She hands you <payment> for the trouble of transport, then says, "Thanks for the shipment. I know that you captains think that we're all bad, but I'm glad that you were able to aid the more friendly among us.`
```

**Pull request merged**